README.md

```diff
@@ -14,12 +14,94 @@ $ npm install @degjs/expander
 
```

Comment:
[suggestion] This doesn't necessarily have to be part of this PR, but it might be helpful to document what accessibility features this plugin provides (aria attributes, etc)